backend/backend.go

```diff
@@ -650,6 +661,33 @@ func (backend *Backend) AccountsTotalBalanceByKeystore() (map[string]KeystoreTot
 	return totalAmounts, nil
 }
 
+// updateTotalWithLightningBalance adds lightning balance to current fiat total.
+func (backend *Backend) updateTotalWithLightningBalance(currentTotal *big.Rat) (*big.Rat, error) {
+	fiat := backend.Config().AppConfig().Backend.MainFiat
+	coinDecimals := new(big.Int).Exp(
+		big.NewInt(10),
+		big.NewInt(int64(8)),
```

Comment:
You can retrieve the BTC coin with `backend.Coin(coinpkg.CodeBTC)` and use it to retrieve the values for decimals and "BTC" unit.

Reply:
I added the `backend.Coin(coinpkg.CodeBTC)` to the `convertAmountToFiat` function, now the function is only for BTC amounts, shall we make it work for other coins?

Reply:
We could make it work with any coin, or rename it to `convertBtcAmountToFiat`. Maybe keep it only on BTC is good for now, we can always extend. Also, generic backend refactoring are not in the scope of the branch, and makes merging the master more difficult